### DISCUSSION TITLE
cannot import name 'mean_iou' from 'torch_geometric.utils'

### DISCUSSION THREAD

**Discussion by Shaun-Wong** [pyg-team/pytorch_geometric#7434] [General]
cannot import name 'mean_iou' from 'torch_geometric.utils'

**Comment by rusty1s** [comment]
We removed metric computation a while ago, and upstreamed them to `torchmetrics`. Which example are you trying to run? You might need to jump back to an earlier PyG version.

**Reply by Spoutnik97** [reply]
I try to use an open source project which use it. How can I replace this function ? I don't find this function in `torchmetrics`

The repo : https://gitlab.com/qub_femg/machine-learning/cadnet/-/tree/master, in the `train.py` file

**Reply by akihironitta** [reply]
I think you should be able to replace it with Jaccard index: https://torchmetrics.readthedocs.io/en/v1.0.0/classification/jaccard_index.html
FYI, the `mean_iou` removed from PyG was this: https://github.com/pyg-team/pytorch_geometric/blob/2.0.4/torch_geometric/utils/metric.py#L176-L201
